lib/controllers/display_controller.c

```diff
@@ -1,15 +1,19 @@
 #include "display_controller.h"
 #include "message.h"
 #include "scheduler.h"
+#include "string.h"
```

Comment:
If this would be in <> to keep up with naming convention would be nice but one place doesn't matter I guess